CMP_CompressonatorLib/DXTC/Codec_DXTC_RGBA.cpp

```diff
@@ -144,29 +144,25 @@ CodecError CCodec_DXTC::CompressRGBBlock(CMP_BYTE rgbBlock[BLOCK_SIZE_4X4X4], CM
 
 CodecError CCodec_DXTC::CompressRGBBlock_Fast(CMP_BYTE rgbBlock[BLOCK_SIZE_4X4X4], CMP_DWORD compressedBlock[2])
 {
-#ifdef _WIN32
-#ifndef DISABLE_TESTCODE
-    DXTCV11CompressBlockSSE((DWORD*) rgbBlock, compressedBlock);
-#else
     this->CompressRGBBlock(rgbBlock, compressedBlock);
-#endif
-#else
-    this->CompressRGBBlock(rgbBlock, compressedBlock);
-#endif
     return CE_OK;
 }
 
 CodecError CCodec_DXTC::CompressRGBBlock_SuperFast(CMP_BYTE rgbBlock[BLOCK_SIZE_4X4X4], CMP_DWORD compressedBlock[2])
 {
-    #if defined(USE_SSE2)
-#ifdef _WIN64
-    // todo: fix sse2 asm function 
-    DXTCV11CompressBlockSSE2((DWORD*)rgbBlock, compressedBlock);
-#else
-    DXTCV11CompressBlockSSEMinimal((DWORD*)rgbBlock, compressedBlock);
-#endif
-    #elif defined(USE_SSE)
-        DXTCV11CompressBlockSSE((DWORD*) rgbBlock, compressedBlock);
+    #ifndef NO_LEGACY_BEHAVIOR
```

Comment:
I'm not sure how to integrate the assembly code that was used here but I'm also not sure if it's still necessary. This is a big `TODO`